Allow users to delete comments.

jmcarp opened the pull request:
h/t @xtine @cmc333333

xtine commented:
Ahhh nice. Last night I made a thing that would remove the comment key if a user tried to save a blank input, but this is more... *clear*. ;D

jmcarp commented:
I see what you did there @xtine. Also, sorry I forgot to make/assign an issue for this--didn't mean to duplicate work!

Also, fixed the linter error I introduced here earlier.

cmc333333 commented:
Two :+1:s make a merge
